integration/skydb_v2_get_set_json.test.ts

```diff
@@ -0,0 +1,240 @@
+import { client, customOptions, dataKey, portal } from ".";
```

Comment:
I would rename this file as it's not about testing get/setJSON, which is still done in `skydb_v2.test.ts`, but data races specifically.

Reply:
renamed 👍